includes/admin/customers/customers.php

```diff
@@ -777,43 +783,58 @@ function edd_customers_view( $customer = null ) {
 				foreach ( $all_emails as $key => $email ) : ?>
 
 					<tr data-key="<?php echo esc_attr( $key ); ?>">
-						<td class="column-actions">
-							<span><?php echo esc_html( $email ); ?></span>
+						<td class="column-primary">
+							<span><?php echo esc_html( $email->email ); ?></span>
 
-							<?php if ( 'primary' === $key ) : ?>
+							<?php if ( 'primary' === $email->type ) : ?>
 								<span class="edd-chip"><?php esc_html_e( 'Primary', 'easy-digital-downloads' ); ?></span>
+							<?php else : ?>
+								<div class="row-actions">
+									<?php
+									$base_url    = edd_get_admin_url(
+										array(
+											'page' => 'edd-customers',
+											'view' => 'overview',
+											'id'   => urlencode( $customer->id ),
+										)
+									);
+									$promote_url = wp_nonce_url( add_query_arg( array( 'email' => rawurlencode( $email->email ), 'edd_action' => 'customer-primary-email' ), $base_url ), 'edd-set-customer-primary-email' );
+									$remove_url  = wp_nonce_url( add_query_arg( array( 'email' => rawurlencode( $email->email ), 'edd_action' => 'customer-remove-email'  ), $base_url ), 'edd-remove-customer-email' );
+									?>
+									<a href="<?php echo esc_url( $promote_url ); ?>"><?php esc_html_e( 'Make Primary', 'easy-digital-downloads' ); ?></a>
+									&nbsp;|&nbsp;
```

Comment:
Small thing -- the extra whitespace makes these a little inconsistent with core row actions:

<img width="336" alt="Screen Shot 2020-07-10 at 1 33 16 PM" src="https://user-images.githubusercontent.com/491124/87182347-fabe5900-c2b1-11ea-8229-3b242b371259.png">